package.json

```diff
@@ -69,18 +69,18 @@
     "gulp-uglify": "^3.0.2",
     "html-validate": "7.5.0",
     "html5shiv": "^3.7.3",
-    "jest": "^29.2.1",
-    "jest-axe": "^6.0.0",
+    "jest": "^29.3.1",
+    "jest-axe": "^7.0.0",
```

Comment:
"Breaking" change because it drops Node.js 10 and 12

Also updates `axe-core` (new rules) but all the checks are passing 👍